README.md

```diff
@@ -46,13 +49,13 @@ There are a few different ways to use the Standards within your project. Which o
 
 ### Download
 
-To use the Draft Web Design Standards on your project, you’ll need to include the CSS and JavaScript files in each HTML page in your project.
+To use the Draft Web Design Standards on your project, you’ll need to include the [CSS (*C*ascading *S*tyle *S*heets)](http://www.w3schools.com/css/css_intro.asp) and JavaScript files in each HTML page or dynamic templates in your project.
```

Comment:
@acolter excellent ideas! I've made some updates here in this commit here: f587930 

Better? Worse? Same?